packages/gatsby-theme-newrelic/README.md

```diff
@@ -236,6 +237,23 @@ documentation.](https://www.gatsbyjs.org/packages/gatsby-plugin-robots-txt/)
 Optional configuration for related resources used in the right rail. Currently
 only `Mdx` nodes are supported.
 
+The related resources component is controlled by specific Frontmatter slugs that
+are defined on a page. By setting the Frontmatter for `resources` and `tags` you can control what is populated in this component.
+
+If no resources or tags are available in the page Frontmatter the component will fallback to using the page title as the search query term and related resources will be auto generated.
+
+The order of priority for populating content in this component is driven by:
```

Comment:
```suggestion
The order of priority for populating content is driven by:
```

Removing `in this component` since really this affects the data layer more than anything. Any component can get access to this data, not just "the component".